.github/workflows/ci-conan.yml

```diff
@@ -179,11 +179,14 @@ jobs:
           cache-name: cache-conan-data
         with:
           path: ~/.conan/data
-          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/metadata.json') }}
```

Comment:
I could not see a `metadata.json` file created in the build tree.  Where is this file supposed to originate from?

Reply:
Maybe Conan does not create it anymore?